demos/rosinstall/src/quittable_listener/package.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0"?>
+<package>
+  <name>quittable_listener</name>
+  <version>0.0.0</version>
+  <description>The listener package</description>
+  <maintainer email="me@example.com">me</maintainer>
+  <license>TODO</license>
```

Comment:
a testing tip, not for now but to take it into account. Usually, we use the word dummy to identify parts of the test that need to be present but affect the test in no way. Instead of TODO, DUMMY would be a better placeholder here, I think.